pisa/stages/osc/layers.py

```diff
@@ -422,7 +413,92 @@ def test_Layers():
 
     logging.info('<< PASS : test_Layers >>')
 
+def test_layers_II():
```

Comment:
Arabic numerals >> Roman numerals; feel free to rename `test_layers` to `test_layers_1` if you want `test_layers_2` to follow some sort of convention